src/test/java/com/serenitydojo/fruitmarket/CatalogTest.java

```diff
@@ -1,30 +1,47 @@
 package com.serenitydojo.fruitmarket;
 
+import com.serenitydojo.Catalog;
+import com.serenitydojo.Fruit;
+import com.serenitydojo.FruitNotAvailable;
 import org.junit.Test;
 
+import java.util.List;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class CatalogTest {
 
     @Test
     public void shouldBeAbleToUpdateTheCurrentPriceOfAFruit() {
 // TODO: Uncomment this code and make it work
-        // Catalog catalog = new Catalog();
-        // catalog.setPriceOf(Apple, 4.00);
-        // assertThat(catalog.getPriceOf(Apple)).isEqualTo(4.00);
+        Catalog catalog = new Catalog();
+        catalog.setPriceOf(Fruit.APPLE, 4.00);
+        double priceofFruit = catalog.getPriceOf(Fruit.APPLE);
```

Comment:
```suggestion
        double priceOfFruit = catalog.getPriceOf(Fruit.APPLE);
```
you have multiple instances of camelCase issues. 
![image](https://user-images.githubusercontent.com/2108984/231462309-0a3641c3-575b-4f48-8ebe-976f8f03f12b.png)
my ide complains about these.